### PR TITLE
Auth w linkedin

### PR DESCRIPTION
**Pull request by hoangr1010**
frontend: 
- basic UI for login
- Get Authorization code from LinkedIn
Backend:
- Route for get user info by LinkedIn authorization code

### CODE REVIEW

**Comment by khangnh107** [comment]
LGTM!

File: server/controllers/AuthController.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        res.status(400).send({
+            message: err.message
+        })
+    }
+}
+
+// HANDLERS
+async function getLinkedInToken(authCode) {
+    try {
+        const response = await axios.post('https://www.linkedin.com/oauth/v2/accessToken', {
```
Comment (khangnh107) [inline server/controllers/AuthController.js:24]: This looks fun

Comment (khangnh107) [inline server/controllers/AuthController.js:24]: This looks fun

File: server/controllers/AuthController.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                'Content-Type': 'application/x-www-form-urlencoded'
+            }
+        });
+        return response.data.access_token;
+    } catch (error) {
+        throw new Error('Failed to get LinkedIn token');
+    }
+}
+
+async function getUserData(token) {
```
Comment (khangnh107) [inline server/controllers/AuthController.js:41]: Why the hell this function?

**Review by khangnh107 — APPROVED** [review]
LGTM!

**Review by lnguyen2693 — APPROVED** [review]
LGTM

**Pull request merged**